rules/auth0_rules/auth0_mfa_risk_assessment_enabled.yml

```diff
@@ -0,0 +1,355 @@
+AnalysisType: rule
+Description: An Auth0 User enabled the mfa risk assessment setting for your organization's tenant.
+DisplayName: "Auth0 MFA Risk Assessment Enabled"
+Enabled: true
+Filename: auth0_mfa_risk_assessment_enabled.py
+Runbook: Assess if this was done by the user for a valid business reason. Be vigilant when enabling this setting as it's in the best security interest for your organization's security posture.
+Severity: High
```

Comment:
```suggestion
Severity: Info
```
I'd classify this as an Info level alert

Reply:
agreed, this is in the notion documentation for this detection as well

Reply:
committing change

Reply:
Nice, thank you.